src/helpers/partnerOrganizations.js

```diff
@@ -3,7 +3,29 @@ const Partner = require('../models/Partner');
 
 exports.create = (payload, credentials) => PartnerOrganization.create({ ...payload, company: credentials.company._id });
 
-exports.list = credentials => PartnerOrganization.find({ company: credentials.company._id }).lean();
+exports.list = async (credentials) => {
+  const partnerOrganizationList = await PartnerOrganization
+    .find({ company: credentials.company._id })
+    .populate({
+      path: 'partners',
+      match: { company: credentials.company._id },
+      populate: { path: 'customerPartners', match: { prescriber: true, company: credentials.company._id } },
+    })
+    .lean();
+
+  const partnerOrganizations = partnerOrganizationList
+    .map(partnerOrganization => (
+      {
+        ...partnerOrganization,
+        prescribedCustomersCount: partnerOrganization.partners
+          .map(partner => partner.customerPartners.length)
+          .flat()
+          .reduce((acc, val) => acc + val, 0),
```

Comment:
je suis pas sûr que le 0 soit nécessaire, normalement reduce((acc, val) => acc + val) suffit

Reply:
Si je ne mets pas de valeur initiale j'ai l'erreur suivante: `TypeError: Reduce of empty array with no initial value`. C'est pour ca que j'ai mis 0